docs/source/style_guide.md

```diff
@@ -0,0 +1,187 @@
+# Documentation style guide
+
+*Audience: documentation writers and editors
+
+While this style guide will also refer to best practices using ReStructured Text (also known as RST), in general we advise writing documentation in Markdown, as it's a more universally accepted documentation standard. Both formats are usable, however, and if you decide to write a topic in RST (or are editing an RST topic), be sure to refer to this style guide.
+
+**When in doubt, use the docs themselves for guidance on how to format things.**
+
+* [For RST formatting](http://hyperledger-fabric.readthedocs.io/en/release-1.4/channel_update_tutorial.html).
+
+* [For Markdown formatting](http://hyperledger-fabric.readthedocs.io/en/release-1.4/peers/peers.html).
+
+Note that you will have to navigate to the Fabric repo to look at the raw file. To do this, click on `Edit on Github` link in the upper right hand corner of the page. Then click the `Raw` tab. This will show you the formatting of the doc.
```

Comment:
When I read this it makes me think we are suggesting they edit the content inline in github. Maybe we are and maybe that is ok? It is just a little ambiguous to me what we are actually recommending here.

Reply:
Yeah, that's a good point. I just didn't want to put people off by starting with all of the steps for cloning. But we do need to make it clear that they shouldn't just try to edit stuff on Guthub.